### PR TITLE
chore(debian): Fix lintian erros and warnings

### LINKED ISSUE #69: Update README.Debian file
---

Author Name: **Mary Laser** 
Original Redmine Issue: 1803, http://www.fossology.org/issues/1803
Original Date: 2012/04/11
Original Assignee: Bob Gobeille

---

http://fossology.svn.sourceforge.net/viewvc/fossology/branches/fossology2.0/fossology/packaging/debian/fossology-common.README.Debian

### PR DESCRIPTION
**Pull request by GMishx**
## Description

Fix some errors and warnings reported by lintian.

### Changes

1. Start executable files in `/usr/share` with a `#!`
    - This effects PHP files from agents:
        1. decider
        1. deciderjob
        1. readmeoss
        1. reportImport
        1. reuser
        1. unifiedreport
        1. spdx2
1. Ignore insane long lines on minified JavaScript files.
1. Fix typos in:
    1. `debian/changelog`
    1. `debian/control`
    1. `debian/fossology-common.README.Debian`
1. Update Wiki link in `install/INSTALL`
1. Update debhelper compatibility level to match Debian Jessie and Ubuntu Xenial Xerus.
1. Split lines > 80 character in `debian/control`.
1. Add aditional meta data to `debian/upstream/metadata` and `debian/watch`.
1. Replace `-s/--same-arch` to `-a/--arch` in `debian/rules`
    - `-s` yields warning: `-s/--same-arch is deprecated; please use -a/--arch instead. This feature will be removed in compat 12.`

## How to test

1. Build a source package for fossology
    1. Run `make tar-release` and change name of the resulting tar to `fossology_3.5.0.orig.tar.gz`.
    1. Copy all files to a new folder without the `.git` folder.
    1. Run `debuild -us -uc -S`.
        1. Check the lintian logs.
1. Build a binary package for fossology
    1. Build `fossology-3.5.0.orig.tar.gz` as mentioned above.
    2. Run `debuild -us -uc -b`.
        1. Check the lintian logs.

    **OR**

    1. Take the source packages built above.
    2. Extract the source using `dpkg-source -x .../path/to/fossology_3.5.0-1.dsc`.
    3. Run `debuild -us -uc -b`.
        1. Check the lintian logs.

# Note
This PR also remove dependency of `fossology-db` from `fossology-web` package.

Closes #69

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+ libmxml-dev, libtext-template-perl, librpm-dev, subversion, rpm, libpcre3-dev,
+ libssl-dev, postgresql-server-dev-all, libboost-regex-dev,
+ libboost-program-options-dev, libjsoncpp-dev, libjson-c-dev,
+ php5-cli|php7.0-cli|php7.2-cli, php-mbstring|php5, php-zip|php5, php-xml|php5
 Standards-Version: 3.9.1
-Homepage: http://fossology.org
+Homepage: https://fossology.org
 
 Package: fossology-dev
 Architecture: any
```
Comment (ag4ums) [inline debian/control]: W: fossology source: debhelper-but-no-misc-depends fossology-dev
need to put misc dependecy

Comment (GMishx) [reply debian/control]: Fixed, please check again.

**Comment by ag4ums** [comment]
@GMishx, please resolve the conflicts

**Comment by GMishx** [comment]
> @GMishx, please resolve the conflicts

@ag4ums done, please check

**Comment by mcjaeger** [comment]
merged because test was OK after restart and the conflicts are also out

**Pull request merged**